src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -26,10 +26,21 @@ private string GetDebuggerDisplay()
         internal const int ListKind = 1;
 
         private readonly ushort _kind;
-        protected NodeFlags flags;
-        private byte _slotCount;
+        private NodeFlagsAndSlotCount _nodeFlagsAndSlotCount;
         private int _fullWidth;
 
+        protected NodeFlags flags
```

Comment:
to keep the impact of this pr low, i kept the names of the fields and hid them behind properties.  If we do want to change these names we can.  Note that `_slotCount` is a challenge as we have both `_slotCount` and `SlotCount` already.  I don't mind keeping these named as fields, as they are effectively still fields, jsut more efficiently stored.